examples/infra/eks-bootstrap/outputs.tf

```diff
@@ -0,0 +1,34 @@
+output "cluster_name" {
+  description = "EKS cluster name"
+  value       = var.cluster_name
+}
+
+output "oidc_issuer" {
```

Comment:
I'm not sure some of these outputs belong here.

I guess because they're mirroring what's in the configmap, but I would probably say the OIDC etc.. is more an output of the module that creates the cluster.

Reply:
I would be tempted to leave outputs for the things this module will create, which I think is the key name and the bastion info.  I can also see an argument for the cluster name as often modules seem to re-output some of their inputs.

Happy deferring a full decision here.